src/Security/Authenticator.php

```diff
@@ -50,6 +52,9 @@ public function onAuthenticationSuccess(Request $request, TokenInterface $token,
         $previousLogin = $user->getDerniereConnexionAt()?->format('Y-m-d');
         $now = new \DateTime();
         $user->setLastIp($request->getClientIp())->setDerniereConnexionAt($now);
+        if ($this->passwordHelper->isStrong($request->request->get('_password'), $user->isBeneficiaire())) {
```

Comment:
`if (!$user->hasPasswordWithLatestPolicy() && $this->passwordHelper->isStrong($request->request->get('_password'), $user->isBeneficiaire()))`
So we don't have to check password strength each time